bootstrap/styles.scss

```diff
@@ -0,0 +1,39 @@
+.navbar {
+  .navbar-brand {
+    color: #878b9b;
+    font-family: Roboto Condensed;
+    font-size: 26px;
+    font-style: normal;
+    font-weight: 700;
+    line-height: normal;
+  }
+}
+.nav-bar {
```

Comment:
there is no such class as **.nav-bar** .  You have class **.navbar** (without dash). If you overwriting bootstrap here would be nice to leave a comment, otherwise, it's confusing.